app/controllers/concerns/providers/steppable.rb

```diff
@@ -33,8 +33,12 @@ module Steppable
         # Back determined by controller action logic
       },
       check_provider_answers: {
-        forward: :providers_legal_aid_applications_path,
+        forward: :providers_legal_aid_application_about_the_financial_assessment_path,
         back: :providers_legal_aid_application_check_benefits_path
+      },
+      about_the_financial_assessments: {
+        forward: :submit_providers_legal_aid_application_about_the_financial_assessment_path,
```

Comment:
`forward` of this new page is not use, is it? If so, maybe it would make more sense to not define it.